integration-cli/docker_cli_ps_test.go

```diff
@@ -539,3 +539,18 @@ func (s *DockerSuite) TestPsFormatMultiNames(c *check.C) {
 	}
 
 }
+
+func (s *DockerSuite) TestPsFormatHeaders(c *check.C) {
+	// make sure no-container "docker ps" still prints the header row
+	out, _ := dockerCmd(c, "ps", "--format", "table {{.ID}}")
```

Comment:
ID results in two words, not two columns -- I went with single column outputs here so that we wouldn't have to deal with potentially inconsistent `TabWriter` output (since it'll make nice spacing so the table is visually "pretty").